src/components/Nav/Nav.js

```diff
@@ -1,8 +1,108 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
+
+class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      scrollTop: 0,
+      isLogin: false,
+      category: ['DINING', 'CAFE', 'CUPBOARD'],
+    };
+  }
+
+  handleScroll = e => {
+    const scrollTop = ('scroll', e.srcElement.scrollingElement.scrollTop);
+    this.setState({
+      scrollTop,
+    });
+  };
+
+  componentWillUnMount = () => {
+    window.removeEventListener('scroll', this.handleScroll);
+  };
+
+  componentDidMount = () => {
+    window.addEventListener('scroll', this.handleScroll);
+
+    if (!localStorage.getItem('userToken')) {
+      this.setState({
+        isLogin: false,
+      });
+    } else {
+      this.setState({
+        isLogin: true,
+      });
+    }
+  };
+
+  handleLog = () => {
+    if (localStorage.getItem('userToken') && this.state.isLogin) {
+      this.setState({
+        isLogin: false,
+      });
+      alert('로그아웃 되었습니다.');
+      localStorage.removeItem('userToken');
+    }
+  };
+
+  test = () => {
+    if (!localStorage.getItem('userToken')) {
+      localStorage.setItem('userToken', 'asd');
+      this.setState({
+        isLogin: true,
+      });
+      alert('로그인 되었습니다.');
+    }
+  };
 
-class Nav extends React.Component {
   render() {
-    return null;
+    const { scrollTop } = this.state;
+    console.log(this.state);
+    console.log(localStorage);
+    return (
+      <>
+        <nav
+          className={!scrollTop ? 'original' : 'original change'}
```

Comment:
템플릿 리터럴 사용해서 다음과 같이 작성할 수 있습니다!

```suggestion
          className={`original ${scrollTop ? 'change' : ''}`}
```